spec/api.md

```diff
@@ -0,0 +1,4019 @@
+                      {
+                        "url": "date",
+                        "valueDate": "2019-08-20"
+                      },
+                      {
+                        "url": "type",
+                        "valueCodeableConcept": {
+                          "coding": [
+                            {
+                              "system": "https://openimis.github.io/openimis_fhir_r4_ig/CodeSystem/contract-premium-type",
+                              "code": "C",
+                              "display": "Cash"
+                            }
+                          ]
+                        }
+                      }
+                    ],
+                    "url": "https://openimis.github.io/openimis_fhir_r4_ig/StructureDefinition/contract-premium"
+                  }
+                ],
+                "typeReference": [
+                  {
+                    "reference": "Patient/44d64808-bca1-4043-89b1-aa045dcc0b04",
+                    "type": "Patient",
+                    "identifier": {
+                      "type": {
+                        "coding": [
+                          {
+                            "system": "https://openimis.github.io/openimis_fhir_r4_ig/CodeSystem/openimis-identifiers",
+                            "code": "UUID"
+                          }
+                        ]
+                      },
+                      "value": "44d64808-bca1-4043-89b1-aa045dcc0b04"
+                    },
+                    "display": "170000001"
+                  }
+                ],
+                "period": [
+                  {
+                    "start": "2019-08-20",
+                    "end": "2020-08-19"
+                  }
+                ],
+                "usePeriod": [
+                  {
+                    "start": "2019-08-20",
+                    "end": "2020-08-19"
+                  }
+                ],
+                "valuedItem": [
+                  {
+                    "entityReference": {
+                      "reference": "InsurancePlan/9ad6e81d-ce42-43ba-aa2e-4ec3978352e8",
+                      "type": "InsurancePlan",
+                      "identifier": {
+                        "type": {
+                          "coding": [
+                            {
+                              "system": "https://openimis.github.io/openimis_fhir_r4_ig/CodeSystem/openimis-identifiers",
+                              "code": "UUID"
+                            }
+                          ]
+                        },
+                        "value": "9ad6e81d-ce42-43ba-aa2e-4ec3978352e8"
+                      },
+                      "display": "BCUL0001"
+                    },
+                    "net": {
+                      "value": 10000.0
+                    }
+                  }
+                ]
+              }
+            ]
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+## Notify payment BB of new payment request
```

Comment:
Would be nice to extend the documentation in this part in accordance with the previous chapters.